onnxscript/rewriter/pattern.py

```diff
@@ -1048,6 +1049,7 @@ def __init__(
             matcher: The pattern matcher that will be used to match the pattern.
                 If not provided, a default matcher will be used.
             verbose: The verbosity level of the rule.
+            name: for debugging purpose
```

Comment:
Would there be a way for formalize this param? Maybe include a recommended way of using it? Something along the lines of "An optional name for the pattern that will show up in verbose logging" etc.

Reply:
done